scripts/azure-pipelines/build.ps1

```diff
@@ -0,0 +1,48 @@
+﻿# Builds Mumble using the given environment and build script.
+# The configuration we build with is adjusted to be close to
+# our release builds.
```

Comment:
What for? So we can for the most part verify our builds here. as closely as possible to our release env?

Reply:
That's correct.